src/main/resources/integrationTestCases/green/singlesample/singlesample.wdl

```diff
@@ -0,0 +1,1661 @@
+## Copyright Broad Institute, 2016
```

Comment:
Yea I'm not sure, this is just the same wdl GOTC uses?  I'm not really sure if it matters to us